print('Exiting')
e = 'error'
exit(e)
print(e)
